src/common/snippets/src/lowered/linear_ir.cpp

```diff
@@ -69,6 +71,8 @@ std::shared_ptr<LinearIR> LinearIR::clone() const {
     // It's Ok to share shapeInfer factory ptr, since the factory doesn't depend on LIR in any way
     cloned->m_shape_infer_factory = m_shape_infer_factory;
     cloned->m_shape_infer = std::make_shared<LIRShapeInfer>(cloned->m_expressions, cloned->m_io_expressions);
+    cloned->m_runtime_configurator = m_runtime_configurator->clone(*cloned);
```

Comment:
It is strange to see that `clone()` take input arguments. If we want to clone an existing class instance, the instance already knows all the field values, so why do we need some arguments?
`clone()` with arguments looks like a constructor to me 🙂